seccomp_default_linux.go

```diff
@@ -317,6 +321,7 @@ func DefaultProfile() *Seccomp {
 				"signalfd",
 				"signalfd4",
 				"sigreturn",
+				"socket",
```

Comment:
I think socket is in here already, but as a conditional rule - it's part of what broke 2.0 CI

Reply:
Those are specific socket calls that allows seccomp and the container engines to catch "AUDIT_WRITE" calls and cause them to change behaviour.  This socket is more generic.  The socket you are talking about does require a newer RUNC for support.  It is fairly recently supported in the runtime specification.

Reply:
I don't think having 2 rules, one conditional and one unconditional, for the same syscall makes sense? Depending on how this is compiled, the conditional call will be below the unconditional call and will never be executed?

Reply:
yes, it must be conditional too, otherwise all socket operations are allowed:

https://github.com/seccomp/containers-golang/pull/27/files#diff-e14a8201925380b28b6d31475b7fbe2cR773-R875

Reply:
How do we make it allowed for all cases other then AUDIT_WRITE?